eclair-core/src/test/scala/fr/acinq/eclair/channel/states/c/WaitForDualFundingConfirmedStateSpec.scala

```diff
@@ -774,14 +814,8 @@ class WaitForDualFundingConfirmedStateSpec extends TestKitBaseClass with Fixture
     bob2 ! INPUT_RECONNECTED(bob2alice.ref, bobInit, aliceInit)
     val bobChannelReestablish = bob2alice.expectMsgType[ChannelReestablish]
     alice2 ! bobChannelReestablish
-    // When reconnecting, we watch confirmation again, otherwise if a transaction was confirmed while we were offline,
-    // we won't be notified again and won't be able to transition to the next state.
-    assert(alice2blockchain.expectMsgType[WatchFundingConfirmed].txId == aliceData.metaCommitments.latest.fundingTxId)
-    aliceData.previousFundingTxs.foreach(f => alice2blockchain.expectMsgType[WatchFundingConfirmed].txId == f.sharedTx.txId)
```

Comment:
Maybe check `alice2blockchain.expectNoMessage(100 millis)` and `bob2blockchain.expectNoMessage(100 millis)`?

Reply:
Done in 6e63a7623.